snuba/query/allocation_policies/per_referrer.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import annotations
+
+import logging
+
+from snuba.query.allocation_policies import (
+    AllocationPolicyConfig,
+    QueryResultOrError,
+    QuotaAllowance,
+)
+from snuba.query.allocation_policies.concurrent_rate_limit import (
+    BaseConcurrentRateLimitAllocationPolicy,
+)
+from snuba.redis import RedisClientKey, get_redis_client
+from snuba.state.rate_limit import RateLimitParameters
+
+DEFAULT_CONCURRENT_QUERIES_LIMIT = 22
+DEFAULT_PER_SECOND_QUERIES_LIMIT = 50
```

Comment:
These values don't seem to be getting used. Is that intentional?

Reply:
nope, I'll remove those

Reply:
Done